.github/workflows/gitbook-release.yaml

```
@@ -0,0 +1,40 @@
+name: Publish Flash SDK and Docs
+run-name: Publish Flash SDK ${{ github.event.release.tag_name }} (by ${{ github.actor }})
+
+on:
+  workflow_dispatch: {}
+  release:
+    types: [published]
+
+jobs:
+  docs:
+    name: Generate and Publish Documentation
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js for Docs
+        uses: actions/setup-node@v4
+        with:
+          cache: yarn
+          node-version-file: .nvmrc
+
+      - name: Install Dependencies
+        run: yarn install --immutable
+
+      - name: Generate Documentation with TypeDoc
+        run: yarn docs:gen-md
+
+      - name: Organize Docs into Version Folder
+        run: |
+          mkdir -p versions/${{ github.event.release.tag_name }}
+          cp -R docs/* versions/${{ github.event.release.tag_name }}
```

consider moving the generated folder instead of copying

```suggestion
          mkdir -p versions
          mv docs versions/${{ github.event.release.tag_name }}
```

---

done

---

done